datasource/context.go

```diff
@@ -165,7 +165,7 @@ func (m ContextUrlValues) Get(key string) (value.Value, bool) {
 		}
 		return value.NewValue(vals), true
 	}
-	return value.EmptyStringValue, false
+	return value.EmptyStringValue, true
```

Comment:
How do you tell the difference between a present key with an empty value and a missing key?